README.md

```diff
@@ -40,3 +40,28 @@ There are two steps for uploading docs:
 
 For an example of using `docuploader`, see
 [example usage in googleapis/google-cloud-go](https://github.com/googleapis/google-cloud-go/blob/main/internal/kokoro/publish_docs.sh).
+
+## Requirements for DocFX YAML tarballs
+
+The tarballs containig DocFX YAML files must adhere to the following requirements:
+
+1. Contains `docs.metadata` or `docs.metadata.json` at the root directory of the
```

Comment:
I assume docs.metadata.json would be a JSON representation... is docs.metadata also expected to be JSON? Is there any difference between them?

Reply:
Okay, I'm still somewhat confused about the requirements then - if it can be in really *any* format, it's odd to have its presence as a requirement. Utterly absurd straw man: my "file format" is a voice recording (in mp3 format) of me dictating the configuration. That would presumably cause problems at the other end... presumably *something* is consuming `docs.metadata` and expecting a certain file format, isn't it?

This doesn't block me, as I'll create docs.metadata.json anyway... but I thought I'd point out the ambiguity.

Reply:
Ah, apologies - I meant to say `any format` between `docs.metadata` or `docs.metadata.json`. That's the only two supported file format we have. I added in a bit to help clarify.

In terms of the difference: most workflows have opted to create `docs.metadata` using `docuploader create-metadata`, but if you want to use less Python and contain the same information in json format we also support that. There's no difference as long as they would contain the same information, only making it easier to adapt to your workflow.

Reply:
It's still not clear to me what format that docs.metadata file *is* in though. Is it the protobuf binary wireformat, YAML, something else? The only description of docs.metadata at the moment is as a filename - that tells me nothing about its content.

Reply:
Ah. Gotcha.

How about:

```
... Either format is accepted, see [`Usage`](#usage) for creating
`docs.metadata` and [`metadata.proto`](https://github.com/googleapis/docuploader/blob/add_requirements/docuploader/protos/metadata.proto) for the metadata format for `docs.metadata`.

Reply:
But that still doesn't say what the format *is*. If it's just "the binary protobuf format" then let's say that. I can draft a suggestion for the text if that's the case, but at the moment all I know is how to make docuploader create the file, rather than what the format of the file is. (And the latter would be more useful.)

Reply:
I'm not sure what the format would be called, would an example link help? Something like [Python example](https://github.com/googleapis/doc-pipeline/blob/29b25c0399aa295db9dfa59fa78214c9a810350d/testdata/python/docs.metadata)

Reply:
Okay, that's protobuf text format. So I'd suggest something like:

> The tarball must contain metadata in the form of the `Metadata` message from metadata.proto. This can be represented in protobuf text format in a file called `docs.metadata`, or in protobuf JSON format in a file called `docs.metadata.json`.